packages/core/src/Parser.ts

```diff
@@ -70,7 +70,12 @@ export class Parser {
         // the HTML tag of the DOM Node. This way we may not support the node
         // but we don't break it either.
         const context = { ...this.currentContext };
-        context.lastParsed = [new VElement(this.currentContext.node.nodeName)];
+        context.parsingMap = createMap([
+            [
+                new VElement(this.currentContext.currentNode.nodeName),
+                this.currentContext.currentNode,
+            ],
+        ]);
```

Comment:
What was the point of this `parsingMap` again ? (I still don't understand the point of `lastParsed` either.)